### PR TITLE
Add Jacoco step in buildPlugin()

### PR DESCRIPTION
**Pull request by stellargo**
CC @oleg-nenashev

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
maybe it should be opt-in, similar to FindBugs & Co?

File: vars/buildPlugin.groovy [hunk 1]
```diff
@@ -172,6 +173,9 @@ def call(Map params = [:]) {
                                 }
                                 checkstyle(cp)
                             }
+                            if (isMaven && archiveJacoco) {
```
Comment (timja) [inline vars/buildPlugin.groovy:176]: does warnings-ng cover this or do both need to be used?

Comment (timja) [reply vars/buildPlugin.groovy:176]: shouldn't this be the code coverage api plugin?
https://plugins.jenkins.io/code-coverage-api/

Comment (uhafner) [reply vars/buildPlugin.groovy:176]: See #153

**Comment by timja** [comment]
replaced by https://github.com/jenkins-infra/pipeline-library/pull/153

**Pull request closed without merging**